geode-lucene/src/main/java/org/apache/geode/cache/lucene/internal/IndexRepositoryFactory.java

```diff
@@ -152,7 +152,7 @@ protected IndexWriter buildIndexWriter(int bucketId, BucketRegion fileAndChunkBu
     return new IndexWriter(dir, config);
   }
 
-  private boolean reindexUserDataRegion(Integer bucketId, PartitionedRegion userRegion,
+  public boolean reindexUserDataRegion(Integer bucketId, PartitionedRegion userRegion,
```

Comment:
There is no need to change the scope to `public`, at least not in this pull request.